content/swiftui/concepts/viewmodifier/terms/tracking/tracking.md

```diff
@@ -0,0 +1,46 @@
+---
+Title: '.tracking()'
+Description: 'Modifies the spacing of the characters in a Text view.'
+Subjects:
+  - 'Mobile Development'
+  - 'iOS'
+Tags:
+  - 'SwiftUI'
+  - 'SwiftUI Modifiers'
+  - 'Xcode'
+CatalogContent:
+  - 'learn-swift'
+  - 'paths/build-ios-apps-with-swiftui'
+---
+
+The **`.tracking()`** modifier method modifies the spacing of the characters in a [`Text`](https://www.codecademy.com/resources/docs/swiftui/views/text) view.
```

Comment:
```suggestion
The **`.tracking()`** modifier method <alters or adjusts> the spacing of the characters in a [`Text`](https://www.codecademy.com/resources/docs/swiftui/views/text) view.
```

Can we change there verb here to one of the two options I provided?

Reply:
I changed it to adjust.